### PR TITLE
class library: event - fix group in grain event

### PR DESCRIPTION
**Pull request by telephon**
Fixes: #3481.

Quick test:

```supercollider

(type:\grain).play;

g = Group(s);

(type:\grain, group: g).play;
```

### CODE REVIEW

**Comment by brianlheim** [comment]
Thanks! Can you please add a unit test for this? (Not familiar with this code so apologies in advance if that's asking a lot)

**Comment by telephon** [comment]
ok, I'll do that.

**Comment by snappizz** [comment]
this is ✔️  for 3.9.1

**Comment by jamshark70** [comment]
We should fix all affected event types in the main class library (from #3481: "This is a bug now in many of the event types").

We get bugs like this by making a global change and testing only one case. We keep bugs like this by fixing them on a case-by-case basis.

**Comment by jamshark70** [comment]
Ok, I did a quick sweep of Event.sc and EventTypesWithCleanup.sc and it seems \grain was the only one missing the conversion. So I think this is good to go. Thanks, Julian!

**Comment by telephon** [comment]
I'll write a test that covers a little more than this case.

File: testsuite/classlibrary/TestEvent.sc [hunk 1]
```diff
@@ -102,7 +102,98 @@ TestEvent : UnitTest {
 
 	}
 
-	test_server_messages {
+	test_server_message_head_type_grain {
+		this.assertEqualServerMessage(\grain, [ 9, \default, -1, 0,  this.defaultGroupID])
```
Comment (snappizz) [inline testsuite/classlibrary/TestEvent.sc]: whitespace: don't put spaces after `[` or before `]`, use exactly one space after `,`

File: testsuite/classlibrary/TestEvent.sc [hunk 2]
```diff
@@ -144,17 +235,32 @@ TestEvent : UnitTest {
...
-	defaultGroup { ^Group.basicNew(nil, 1) }
+
+	latency { ^Server.default.latency }
+	defaultGroupID { ^Server.default.defaultGroupID }
+	defaultGroup { ^Server.default.defaultGroup }
+
+
+
+
+
```
Comment (snappizz) [inline testsuite/classlibrary/TestEvent.sc]: too many blank lines

File: testsuite/classlibrary/TestEvent.sc (hunk 1 above)
Comment (telephon) [reply testsuite/classlibrary/TestEvent.sc]: I didn't put them there personally, but I can remove them ;)

**Pull request merged**